examples/tutorial/02-email-copilot/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# This is a simplified example for learning purposes. Do not use this in production.
+# For production-ready deployments, see: https://www.tensorzero.com/docs/gateway/deployment
+
+services:
+  clickhouse:
+    image: clickhouse/clickhouse-server
+    ports:
+      - "8123:8123"
+    healthcheck:
+      test: wget --spider --tries 1 http://localhost:8123/ping
+      start_period: 30s
+      start_interval: 1s
+      timeout: 1s
+
+  gateway:
+    image: tensorzero/gateway:2024.12.1
```

Comment:
are we sure we wanna pin the version here? I think that's gonna be a pain to maintain

Reply:
mistake, that was to circumvent the issue earlier